brozzler/ydl.py

```diff
@@ -32,6 +33,33 @@
 thread_local = threading.local()
 
 
+def is_html_maybe(url):
+    if "chrome-error:" in url:
+        return False
+
+    skip_url_exts = ["pdf", "jpg", "jpeg", "png", "gif", "mp3", "mp4", "mpeg", "css", "js"]
```

Comment:
This list of extensions is really limited. I suggest that you do something like the following outside the function and then use `EXTENSIONS` inside it.
```
import mimetypes
EXTENSIONS = list(mimetypes.types_map) + list(mimetypes.common_types)
EXTENSIONS.remove('.htm')
EXTENSIONS.remove('.html')
```